traits.yml

```diff
@@ -146,3 +267,58 @@ traits:
             type: string
             description: >
               Contents of the token.
+specifications:
+  lifecycle:
+    description:
+      Specifications that are concerned with describing aspects of the
+      lifecycle of an entity, such as versioning.
+    members:
+      EntityVersionsRelationship:
+        description: >
+          Retrieves references to alternate versions of the same logical
+          entity. This may include unstable versions such as "latest".
+        traitSet:
+          - namespace: usage
+            name: Relationship
+          - namespace: relationship
+            name: Unbounded
+          - namespace: lifecycle
+            name: Version
```

Comment:
[Plan] Does anyone else wish for a nice syntax for this. Something like 
```
traitSet:
  - name: usage::Relationship
```
worth planning to do eventually?